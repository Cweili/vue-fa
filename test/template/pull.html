<p>
  <fa :icon="faQuoteLeft" pull="left" size="2x"/>
  <fa :icon="faQuoteRight" pull="right" size="2x"/>
</p>
