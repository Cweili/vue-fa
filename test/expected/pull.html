<p><svg viewbox="0 0 512 512" class="" style="height:1em;overflow:visible;vertical-align:-.125em;float:left;font-size:2em;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg">
    <g transform="translate(256 256)" transform-origin="128 0">
      <g transform>
        <path d="M464 256h-80v-64c0-35.3 28.7-64 64-64h8c13.3 0 24-10.7 24-24V56c0-13.3-10.7-24-24-24h-8c-88.4 0-160 71.6-160 160v240c0 26.5 21.5 48 48 48h128c26.5 0 48-21.5 48-48V304c0-26.5-21.5-48-48-48zm-288 0H96v-64c0-35.3 28.7-64 64-64h8c13.3 0 24-10.7 24-24V56c0-13.3-10.7-24-24-24h-8C71.6 32 0 103.6 0 192v240c0 26.5 21.5 48 48 48h128c26.5 0 48-21.5 48-48V304c0-26.5-21.5-48-48-48z" fill="currentColor" transform="translate(-256 -256)"></path>
      </g>
    </g>
  </svg><svg viewbox="0 0 512 512" class="" style="height:1em;overflow:visible;vertical-align:-.125em;float:right;font-size:2em;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg">
    <g transform="translate(256 256)" transform-origin="128 0">
      <g transform>
        <path d="M464 32H336c-26.5 0-48 21.5-48 48v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48zm-288 0H48C21.5 32 0 53.5 0 80v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48z" fill="currentColor" transform="translate(-256 -256)"></path>
      </g>
    </g>
  </svg></p>